packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -3,11 +3,27 @@ Cypress.Commands.add("startWildflyContainer", () => {
     "start:wildfly:container",
     {
       name: Cypress.spec.name.replace(/\.cy\.ts/g, "").replace(/-/g, "_"),
+      configuration: "standalone-insecure.xml",
+      useNetworkHostMode: false,
     },
     { timeout: 240_000 }
   );
 });
 
+Cypress.Commands.add("startWildflyContainerSecured", (configuration) => {
+  return cy.task("start:wildfly:container", {
+    name: Cypress.spec.name.replace(/\.cy\.ts/g, "").replace(/-/g, "_"),
+    configuration: configuration,
```

Comment:
Why do we need the configuration parameter? Can we use `"standalne.xml"` here instead?

Reply:
hardcode the configuration file name